modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/GetContentByIdCommand.java

```diff
@@ -29,7 +30,7 @@ Content execute()
             final Node node = nodeService.getById( nodeId );
             content = filter( translator.fromNode( node, true ) );
         }
-        catch ( NodeNotFoundException e )
+        catch ( NodeNotFoundException | ContentNotFoundException e )
```

Comment:
I think will be better return `null` instead of throw `ContentNotFoundException` in the translator.